blog/invictus-logic-apps-testing-framework.md

```diff
@@ -0,0 +1,118 @@
+Announcing the Invictus Test Framework for Azure Logic Apps
+--
+>While writing unit or integration tests should be part of every project, this proofs to be quite difficult whenever creating integration scenarios using Azure Logic Apps.
+>Thanks to the creation of the Invictus Test Framework for Azure Logic Apps, there is no longer any excuse not to write any test-cases.
+
+Up until now, when building interfaces on top of Azure Logic Apps, you were forced to manually (re)test every implemented feature because there simply wasn't anything available to help you on your way, which ends up costing both time and money you'd like to spend elsewhere.  
```

Comment:
```suggestion
Up until now, when building interfaces on top of Azure Logic Apps, you were forced to manually (re)test every implemented feature because there simply wasn't anything available to help you on your way.

While this approach works fine, we've found that it has a few flaws:
- New people on your team don't have an easy way to understand how flows should work
- It's cumbersome since you cannot easily automate it to guarantee quality
- It takes time & effort which ends up in a higher engineering & testing cost, money you'd like to spend elsewhere
```